lib/origen/application/lsf_manager.rb

```diff
@@ -209,7 +209,7 @@ def clear(options)
           if options[:type] == :all
             File.delete(remote_jobs_file) if File.exist?(remote_jobs_file)
             @remote_jobs = {}
-            return
```

Comment:
This diff is caused by [this](https://github.com/Origen-SDK/origen/blob/d1143f82d2019202b06fa3bec2e6a5b0dc102878/config/rubocop/strict.yml#L4594) rule. Rubocop doesn't indicate that the rule has been updated.

We have this rule [enabled](https://github.com/Origen-SDK/origen/blob/e531b9c03573a6417a4588ca520bcafa8aa41eb8/config/rubocop/strict_enabled.yml#L392-L395) in our existing rules.

Most of the diffs have a similar story.